test/automl/test_forecast.py

```diff
@@ -441,9 +441,137 @@ def test_forecast_classification(budget=5):
     # plt.show()
 
 
+def get_stalliion_data():
+    from pytorch_forecasting.data.examples import get_stallion_data
+
+    data = get_stallion_data()
+    # add time index
+    data["time_idx"] = data["date"].dt.year * 12 + data["date"].dt.month
+    data["time_idx"] -= data["time_idx"].min()
+    # add additional features
+    data["month"] = data.date.dt.month.astype(str).astype(
+        "category"
+    )  # categories have be strings
```

Comment:
have to?

Reply:
no that's just based on pytorch forecast's example.